ogc/features/url.go

```diff
@@ -150,6 +152,9 @@ func (fc featureCollectionURL) validateNoUnknownParams() error {
 	copyParams.Del(bboxCrsParam)
 	copyParams.Del(filterParam)
 	copyParams.Del(filterCrsParam)
+	for _, pf := range fc.configuredPropertyFilters {
+		copyParams.Del(pf.Name)
+	}
 	if len(copyParams) > 0 {
 		return fmt.Errorf("unknown query parameter(s) found: %v", copyParams.Encode())
```

Comment:
beetje ver gezocht, maar misschien de user input truncaten? als iemand een grote input levert moeten wij ook een grote output leveren. voor ddos etc. mwa, beetje ver gezocht, het is niet amplifying

Reply:
done